src/software/thunderscope/robot_diagnostics/chicker.py

```diff
@@ -73,6 +95,86 @@ def __init__(self, proto_unix_io):
         self.geneva_value = 3
         self.power_value = 1
 
+    def kick_clicked(self):
+        # if button is enabled
+        if self.kick_button_enable:
+            # send kick primitive
+            self.send_command(ChickerCommandMode.KICK)
+            self.toggle_kick_button()
+
+            # set and start timer to re-enable kick button after 3 seconds
+            self.start_timer_once(self.toggle_kick_button, 3 * MILLISECONDS_PER_SECOND)
```

Comment:
As per the above conversation with Cody, I think the desired bahviour is that the timer is shared for kicking and chipping (i.e. if I kick, I shouldn't be able to chip within the next 3 seconds as well)